wikimedia/wikipedia-ios
# Watchlist accessibility improvements

**Phabricator:** https://phabricator.wikimedia.org/T342359

### Notes
* This PR adds some accessibility adjustments for voiceOver users to make the watchlist feature easier to navigate
* The main difference is that when VoiceOver is running, tapping on the cell doesn't take you directly to the diff. Instead, it opens the menu with "Go to Diff" as the first action. That was done to make it less confusing to VoiceOver users and to concentrate all actions in one.

### Test Steps
1. With VoiceOver on, access the Watchlist Feature from Article. Add an article to the watchlist, choose the expiration date, and ensure the toast is accessible.
2. Access the watchlist from Account in Settings, make sure it's easy to navigate.
3. Try it on a fresh install so the onboarding modal shows
